src/main/java/ee/buerokratt/ruuter/configuration/ApplicationProperties.java

```diff
@@ -14,7 +14,7 @@ public class ApplicationProperties {
     private String configPath;
     private Boolean stopInCaseOfException;
     private List<Integer> httpCodesAllowList;
-    private DefaultAction defaultAction = new DefaultAction();
+    private HttpDefaultService defaultServiceInCaseOfException = new HttpDefaultService();
```

Comment:
Lets rename `service` to `configuration` (all other DSL's are named as "confs")

Reply:
all "service"/"configuration" will be refactored to "dsl" in future task